python/python_fastapi_server/configure_logging.py

```diff
@@ -3,7 +3,17 @@
 """Configures logging for the adaguc-server python wrapper"""
 import sys
 
+def run_once(f):
+    def wrapper(*args, **kwargs):
+        if not wrapper.has_run:
+            wrapper.has_run = True
+            return f(*args, **kwargs)
 
+    wrapper.has_run = False
+    return wrapper
+
+
+@run_once
 def configure_logging(logging):
```

Comment:
Not clear to me why it is called twice. The solution is a bit... brutal.

Reply:
I found that gunicorn or uvicorn also add a logging handler.

I tried to add logging.getLogger().handlers.clear() in configure_logging.

This looks like a better solution to me?